nix/cabal-project.nix

```diff
@@ -77,14 +74,14 @@ let
     # Else, we'll get the error
     # `/nix/store/ls0ky8x6zi3fkxrv7n4vs4x9czcqh1pb-marconi/marconi/test/configuration.yaml: openFile: does not exist (No such file or directory)`
     marconi-chain-index.preCheck = "
-      export CARDANO_CLI=${inputs.cardano-node.legacyPackages.cardano-cli}/bin/cardano-cli${pkgs.stdenv.hostPlatform.extensions.executable}
-      export CARDANO_NODE=${inputs.cardano-node.legacyPackages.cardano-node}/bin/cardano-node${pkgs.stdenv.hostPlatform.extensions.executable}
+      export CARDANO_CLI=${inputs'.cardano-node.legacyPackages.cardano-cli}/bin/cardano-cli${pkgs.stdenv.hostPlatform.extensions.executable}
+      export CARDANO_NODE=${inputs'.cardano-node.legacyPackages.cardano-node}/bin/cardano-node${pkgs.stdenv.hostPlatform.extensions.executable}
       export CARDANO_NODE_SRC=${../.}
     ";
 
     # Needed for running the marconi-sidechain integration tests in CI
     marconi-sidechain.preCheck = "
-      export MARCONI_SIDECHAIN=${inputs.self.packages.marconi-sidechain-exe-marconi-sidechain-ghc927}/bin/marconi-sidechain
+      export MARCONI_SIDECHAIN=${inputs.self.packages.marconi-sidechain}/bin/marconi-sidechain
```

Comment:
Aliases for your cabal executables only (this excludes benchmarks and test suites) are added to your packages and apps flake outputs.
Alias means just the executable name as found in your cabal file, without the `$CabalPackageName-exe-` prefix.

For projects supporting a single compiler (like in this repo, ghc927), the compiler name is no longer appended to the flake outputs names.

inputs'.self.packages.marconi-sidechain-exe-marconi-sidechain-ghc927 ->
inputs'.self.packages.marconi-sidechain

inputs'.self.apps.marconi-sidechain-exe-marconi-sidechain-ghc927 ->
inputs'.self.apps.marconi-sidechain
```
If you cabal project has multiple executables with the same name across your cabal files, a warning will be generated when running `list-flake-outputs` inside `nix develop`, and no aliases will be created.

Reply:
FYI @koslambrou @kayvank